quetz/config.py

```diff
@@ -148,16 +148,22 @@ class Config:
     def __new__(cls, deployment_config: str = None):
         if not deployment_config and None in cls._instances:
             return cls._instances[None]
-        path = os.path.abspath(cls.find_file(deployment_config))
-        if path not in cls._instances:
-            config = super().__new__(cls)
-            config.init(path)
-            cls._instances[path] = config
-            # optimization - for default config path we also store the instance
-            # under None key
-            if not deployment_config:
-                cls._instances[None] = config
-        return cls._instances[path]
+        try:
+            path = os.path.abspath(cls.find_file(deployment_config))
```

Comment:
I think it would be enough to wrap the `try: except:` block only around this line

Reply:
That wouldn't work. Since if it fails, `path` would be undefined and then the rest of the code block wouldn't work.

Reply:
Thanks, somehow it didn't work earlier, but now it does. Probably because I wasn't in the `deployment` directory. I have pushed the changes.